cpg_workflows/workflow.py

```diff
@@ -1270,13 +1272,6 @@ def deprecated_queue_for_cohort(self, cohort: Cohort) -> dict[str, StageOutput |
                 continue
 
             logging.info(f'Dataset {dataset}:')
-            # collect all expected outputs across all samples
-            # find all directories which will be checked
-            # list outputs in advance
-            all_outputs: set[Path] = set()
-            for sequencing_group in dataset.get_sequencing_groups():
-                all_outputs = path_walk(self.expected_outputs(sequencing_group), all_outputs)
-
```

Comment:
this usage of 'path_walk' was designed to collect all the expected outputs across the whole cohort, then use some kind of cached iterator to see which of those files were missing, reducing our overall number of GCP existence checks. That whole process was put in the bin when @illusional adapted the `exists()` call to be a much more elegant cached check. This was looping over all the expected_outputs, storing them in an object, and... not doing anything with it.